fixtures/all-versions.github

```diff
@@ -406,7 +406,7 @@ jobs:
           chmod a+x $HOME/.cabal/bin/cabal-plan
           cabal-plan --version
       - name: checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
this action uses node-16 as per changelog. Does old settinngs for `distribution` work with it?

Reply:
Frankly, I don't know whether this works with `distribution`.
In the long run, we will have to upgrade as `node-12` is deprecated.
Also, `dependabot` is harrassing me about outdated actions:
- https://github.com/agda/fix-whitespace/pull/39